tests/h/services/search_index/_queue_test.py

```diff
@@ -351,35 +338,28 @@ def search_index(
         )
 
     @pytest.fixture
-    def with_indexed_annotations(self, index, annotations):
-        index(annotations)
-
-    @pytest.fixture
-    def with_queued_annotations(self, add_all):
-        add_all()
-
-    @pytest.fixture
-    def index(self, es_client, search_index, nipsa_service):
-        """A function for adding annotations to Elasticsearch."""
-
-        def index(annotations):
-            """Add `annotations` to the Elasticsearch index."""
-            for annotation in annotations:
-                search_index.add_annotation(annotation)
+    def index(self, es_client, search_index):
+        """A method that indexes the given annotation into Elasticsearch."""
 
+        def index(annotation):
+            search_index.add_annotation(annotation)
             es_client.conn.indices.refresh(index=es_client.index)
 
         return index
 
-    @pytest.fixture
-    def add_all(self, queue, annotation_ids):
-        def add_all(ids=annotation_ids, schedule_in=MINUS_5_MIN_IN_SECS, force=False):
-            for id_ in ids:
-                queue.add_by_id(
-                    id_, tag="test_tag", schedule_in=schedule_in, force=force
-                )
+    @pytest.fixture(autouse=True)
+    def noise_annotations(self, factories, index):
+        # Create some noise annotations in the DB. Some of them also in
+        # Elasticsearch, some not. None of these should ever be touched by the
+        # sync() method in these tests.
+        annotations = factories.Annotation.create_batch(size=2)
+        index(annotations[0])
 
-        return add_all
+    @pytest.fixture(autouse=True)
+    def noise_jobs(self, factories):
+        # Create some noise jobs in the DB. None of these should ever be
+        # touched by the sync() method in these tests.
+        factories.Job()
```

Comment:
I realised that because there are no _other_ annotations in Elasticsearch it was possible to break the code and have the tests still pass. Added some noise jobs as well for good measure.